spinn_front_end_common/abstract_models/abstract_data_specable_vertex.py

```diff
@@ -105,12 +105,12 @@ def set_no_machine_time_steps(self, new_no_machine_time_steps):
         :param new_no_machine_time_steps:
         :return:
         """
```

Comment:
Just a note for the future. instead of commenting out the lines, just delete them. github contains the history of the files, and merges such as this show the changes anyhow. Leaving commented out code in a large software stack can open us up to bloated code and makes code less maintainable as you then have to traverse the blocks of uncommented code to find what bits are actually being ran.